apps/src/StudioApp.js

```diff
@@ -1323,6 +1323,48 @@ StudioApp.prototype.onMouseMoveVizResizeBar = function (event) {
   this.resizeVisualization(newVizWidth);
 };
 
+StudioApp.prototype.clearVisualizationSize = function () {
+  if ($('#visualizationColumn').hasClass('wireframeShare')) {
+    return;
+  }
+
+  const editorColumn = $(".editor-column");
+  const visualization = document.getElementById('visualization');
+  const visualizationResizeBar = document.getElementById('visualizationResizeBar');
+  const visualizationColumn = document.getElementById('visualizationColumn');
+
+  if (this.isRtl()) {
+    visualizationResizeBar.style.right = '';
+    editorColumn.css('right', '');
+  } else {
+    visualizationResizeBar.style.left = '';
+    editorColumn.css('left', '');
+  }
+
+  visualizationResizeBar.style.lineHeight = '';
+  visualizationColumn.style.maxWidth = '';
+  visualization.style.maxWidth = '';
+  visualization.style.maxHeight = '';
+
+  if (!utils.browserSupportsCssMedia()) {
+    visualization.style.height = '';
+    visualization.style.width = '';
+  }
+
+  applyTransformScaleToChildren(visualization, '');
+
+  $('#soft-buttons').removeClass('soft-buttons-compact');
+
+  const smallFooter = document.querySelector('#page-small-footer .small-footer-base');
+  if (smallFooter) {
+    smallFooter.style.maxWidth = '';
+    const smallPrint = smallFooter.querySelector('small');
+    smallPrint.style.float = '';
+  }
+
+  utils.fireResizeEvent();
```

Comment:
It looks like in order to take our html screenshot, we end up making a whole bunch of DOM manipulations. I'm somewhat wary of all this, as it feels rather brittle. Would it be possible to instead clone the DOM, make changes in the clone, and then call html2canvas on the clone?

Our long term goal is to move more stuff towards React-like patterns. This feels very non-declarative/non-React like. It also feels like it would it a much more difficult future endeavor to make some of these components (i.e. VisualizationColumn and the like) pure React.

With how many different individual elements we're touching here, I also worry that it would be pretty easy for someone in the future to make a small change to one of these elements that breaks screenshots, and we don't realize (because nothing failed, except that we now ended up with some distorted screenshot).

Thoughts?